amida-tech/hl7-telescoper-api-server
# Hl7 26 po unique uploads

### Related JIRA tickets:
>Please enter the whole URL so we can just click/copy it
- https://jira.amida.com/browse/HL7-26

### What exactly does this PR do?
>i.e. Added logic to do the thing because the thing didn't exist before. Please provide as much detail as possible. 
- Added logic to prevent uploading file with the same name

### What else was added outside of the scope of the ask?
>i.e. Simple bugfix, corrected typos, cleaned up code, etc
- Added a changelog file
- Moved the jwt exp time to as a config var

### Do ANY of these changes introduce a breaking change? (in-scope or otherwise)
>i.e. Users need to update their `.env` files with the following... etc
- Yes, please add 
   - `JWT_EXP_TIME`  and
   - `FILE_UPLOAD_PATH`

### Manual test cases?
>Steps to manually test introduced changes can go here.
>Remember the prerequisites!
>Remember edge-cases you've caught in your code, etc..
> i.e. ignoring the button and clicking on the "NEXT" button should trigger a warning event because the thing that clicking the new button does didn't happen
- Upload a file (see that it saves to the FS as the user collection
- Edit the file if you like (leaving the filename the same) then re-upload that file. You should get a conflicting error.
- Check that the file doesn't get saved

### Any additional context/background?
>N/A if this is straight-forward
- N/A

### Screenshots (if appropriate):
